sugaroid/server/main.py

```diff
@@ -51,7 +51,7 @@ async def brain(websocket, path):
 
 
 async def ws():
-    async with websockets.serve(brain, "localhost", int(os.getenv("PORT", 8000))):
+    async with websockets.serve(brain, "0.0.0.0", int(os.getenv("PORT", 5000))):
```

Comment:
Why switch from `localhost` to `0.0.0.0`? Given they're both same and the variable is a better reference.

Reply:
I think they are actually not the same, or maybe I am not sure. When I tried localhost, which is basically 127.0.0.1, the websocket port was only accessible from the same server, but not outside the server (I use [heroku](https://heroku.com) for deployments), but yea, I think you are right, making it configurable would be way better than hardcoding the value. Thanks for your suggestion!, I will get to working on it :smile:

Reply:
Yes you're right, they're not the same. The websocket behaviour when used with localhost is expected as it's basically talking to itself. Making it configurable is better and besides hardcoding not being good, `0.0.0.0` doesn't point to any particular address but it's probably assigned by DHCP since it's on a server.

Reply:
Agreed, you are right. I made it configurable using the `ADDR` environment variable, which defaults to `localhost` if unset in https://github.com/sugaroidbot/sugaroid/pull/136/commits/c0e462183e73e0b68ac73c14df6f39078dda2b76 and added configuration documentation in https://github.com/sugaroidbot/sugaroid/pull/136/commits/294ae9ba591d321d0e11a341d12af3e69fd9b5f3. Thanks for your review! :rocket: